app/assets/stylesheets/kaui/account.scss

```diff
@@ -1,3 +1,7 @@
+$smphone: unquote("max-width: 450px");
```

Comment:
Interesting that we need that in each file

Reply:
I propose we revisit the entire CSS later this year.